### PR TITLE
GT-123 Update example READ API output

### LINKED ISSUE #12: Update example READ API output
Following on from the refresh of the main GOCDB sample data, see: https://github.com/GOCDB/gocdb/issues/384, the example API output used in the [docs](https://github.com/GOCDB/GOCDB.github.io/tree/main/api/read) should be updated to reflect the new sample data.

This will help improve consistency in the example output betwen the pages and will also make it easier to (re)generate in future when needed.

### PR DESCRIPTION
**Pull request by rowan04**
Resolves #12 

~~Waiting on gocdb/gocdb#394~~

### CODE REVIEW

File: api/read/get_service/index.md [hunk 1]
```diff
@@ -39,38 +39,38 @@ This method does not support paging.
...
 <results>
-  <SERVICE_ENDPOINT PRIMARY_KEY="4180G0">
-    <PRIMARY_KEY>4180G0</PRIMARY_KEY>
-    <HOSTNAME>gocdb.example.com</HOSTNAME>
-    <GOCDB_PORTAL_URL>/portal/index.php?Page_Type=Service&id=4180</GOCDB_PORTAL_URL>
-    <HOSTDN>/DC=org/DC=terena/DC=tcs/C=NL/L=Amsterdam/O=Stichting EGI/gocdb.example.com</HOSTDN>
+  <SERVICE_ENDPOINT PRIMARY_KEY="3303G0">
+    <PRIMARY_KEY>3303G0</PRIMARY_KEY>
+    <HOSTNAME>bdii.ipb.ac.rs</HOSTNAME>
+    <GOCDB_PORTAL_URL>/portal/index.php?Page_Type=Service&id=3303</GOCDB_PORTAL_URL>
```
Comment (tofu-rocketry) [inline api/read/get_service/index.md:45]: Possibly escape the ampersand?

File: api/read/get_roc_contacts/index.md [hunk 2]
```diff
@@ -26,31 +26,31 @@ This method optionally supports paging, to iterate over all results,
...
-    <ROCNAME>EGI.eu</ROCNAME>
-    <MAIL_CONTACT>example@egi.eu</MAIL_CONTACT>
-    <GOCDB_PORTAL_URL>/portal/index.php?Page_Type=NGI&id=4</GOCDB_PORTAL_URL>
-    <CONTACT USER_ID="406G0" PRIMARY_KEY="406G0">
-      <FORENAME>Example</FORENAME>
-      <SURNAME>User</SURNAME>
+  <ROC ID="11463" ROC_NAME="NGI_SI">
+    <ROCNAME>NGI_SI</ROCNAME>
+    <MAIL_CONTACT>ngi-si-contact@swing-grid.si</MAIL_CONTACT>
+    <GOCDB_PORTAL_URL>/portal/index.php?Page_Type=NGI&id=11463</GOCDB_PORTAL_URL>
```
Comment (tofu-rocketry) [inline api/read/get_roc_contacts/index.md:32]: Not actually sure how important it is to escape the ampersands, but in some places they are and GitHub is highlighting them in red.

Comment (gregcorbett) [reply api/read/get_roc_contacts/index.md]: They should be escaped as GOCDB escapes them in it's API output (and indeed an unescaped `&` is invalid XML it seems).

Comment (rowan04) [reply api/read/get_roc_contacts/index.md]: I have escaped them in my latest commit, thanks.

**Comment by gregcorbett** [comment]
Once GOCDB/GOCDB#394 is merged, as second pass of this will be needed to ensure each method's example XML API output reflects the new sample data - that will include escaping aditional `&`, see https://github.com/GOCDB/GOCDB.github.io/pull/14#discussion_r1060693165.

**Comment by rowan04** [comment]
> Once [GOCDB/gocdb#394](https://github.com/GOCDB/gocdb/pull/394) is merged, as second pass of this will be needed to ensure each method's example XML API output reflects the new sample data - that will include escaping aditional `&`, see [#14 (comment)](https://github.com/GOCDB/GOCDB.github.io/pull/14#discussion_r1060693165).

394 has now been merged, what needs doing to this?

**Comment by gregcorbett** [comment]
For each of the pages under [api/read/](https://github.com/GOCDB/GOCDB.github.io/tree/main/api/read/), can you update the example API output so it is a copy/paste from the output of the relevent API endpoint on an instance loaded with the sample data?

It will be sufficent to only include one result in the example API output, i.e. for `get_user` it will be sufficent to only have one `EGEE_USER ID` block.

Some endpoints wont have data populated by the sample data, such as `get_cert_status_changes`. I think for now it's good enough to leave the exmaple API output as is, but note which endpoints this affects.

**Comment by rowan04** [comment]
more ampersands that need escaping :/

**Comment by rowan04** [comment]
> more ampersands that need escaping :/

done

File: api/read/get_cert_status_changes/index.md [hunk 3]
```diff
@@ -31,10 +31,10 @@ This method optionally supports paging, to iterate over all results,
   <result ID="1">
     <TIME>13-Sep-2022 09.21.26</TIME>
     <UNIX_TIME>1663060886</UNIX_TIME>
-    <SITE>GRIDOPS-GOCDB</SITE>
+    <SITE>Izola MRF</SITE>
     <OLD_STATUS>Uncertified</OLD_STATUS>
     <NEW_STATUS>Certified</NEW_STATUS>
-    <CHANGED_BY>/C=UK/O=eScience/OU=CLRC/L=RAL/CN=example user</CHANGED_BY>
+    <CHANGED_BY>/C=UK/O=eScience/OU=CLRC/L=RAL/CN=Rowan Moss</CHANGED_BY>
```
Comment (gregcorbett) [inline api/read/get_cert_status_changes/index.md:37]: Can you change this to one of the fake users?

File: api/read/get_service_types/index.md [hunk 4]
```diff
@@ -23,9 +23,9 @@ This method does not support paging.
 ```xml
 <?xml version="1.0" encoding="UTF-8"?>
 <results>
-  <SERVICE_TYPE TYPE_ID="44G0" PRIMARY_KEY="44G0">
-    <SERVICE_TYPE_NAME>egi.GOCDB</SERVICE_TYPE_NAME>
-    <SERVICE_TYPE_DESC>[Central Service] The central GOCDB</SERVICE_TYPE_DESC>
+  <SERVICE_TYPE TYPE_ID="1G0" PRIMARY_KEY="1G0">
+    <SERVICE_TYPE_NAME>CE</SERVICE_TYPE_NAME>
+    <SERVICE_TYPE_DESC>[Site service] The LCG Compute Element. Currently the standard CE within the gLite middleware stack. Soon to be replaced by the CREAM CE. </SERVICE_TYPE_DESC>
```
Comment (gregcorbett) [inline api/read/get_service_types/index.md:28]: Where does this description come from?

**Review by gregcorbett — CHANGES_REQUESTED** [review]
See comments.

File: api/read/get_cert_status_changes/index.md (hunk 3 above)
Comment (rowan04) [reply api/read/get_cert_status_changes/index.md:37]: ok

File: api/read/get_service_types/index.md (hunk 4 above)
Comment (rowan04) [reply api/read/get_service_types/index.md:28]: from the api output for get_service_types? that was the first service_type output in that api output

Comment (gregcorbett) [reply api/read/get_service_types/index.md:28]: Cool, I couldn't see the corresponding description in the codebase, but maybe I wasn't looking hard enough 😄

**Comment by gregcorbett** [comment]
LGTM, can you merge the new commit into the one that introduced your DN to `api/read/get_cert_status_changes/index.md` (I _think_ it's this commit: d625ccd870023b6c7d625706b0dd01e76551aa34)

**Comment by rowan04** [comment]
> LGTM, can you squash the new commit into the one that introduced your DN to `api/read/get_cert_status_changes/index.md` (I _think_ it's this commit: [d625ccd](https://github.com/GOCDB/GOCDB.github.io/commit/d625ccd870023b6c7d625706b0dd01e76551aa34))

done

**Comment by gregcorbett** [comment]
Awesome, thanks 😄

**Review by gregcorbett — APPROVED** [review]

**Pull request merged**